autoPyTorch/pipeline/components/setup/preprocessor/Preprocessing.py

```diff
@@ -0,0 +1,69 @@
+from typing import Any, Dict, Optional
+
+from ConfigSpace.configuration_space import ConfigurationSpace
+
+import numpy as np
+
+from autoPyTorch.pipeline.components.setup.base_setup import autoPyTorchSetupComponent
+from autoPyTorch.pipeline.components.setup.preprocessor.utils import get_preprocess_transforms, preprocess
+
+
+class Preprocessing(autoPyTorchSetupComponent):
```

Comment:
Do you think you can change the name to something that indicates that preprocessing may happen here?
Ideas:

- EarlyPreprocessing
-PrecedentPreprocessing

Yeah or something nice, but that indicates that this class will do an early preprocessing for efficiency :)